### PR TITLE
Add Clerk

### LINKED ISSUE #9379: Icon request: Clerk
### Brand Name

Clerk

### Website

https://clerk.com/

### Popularity Metric

Similarweb rank: 97,051
https://www.similarweb.com/website/clerk.com/#overview

### Official Resources for Icon and Color

svg: https://clerk.com/_next/image?url=%2Fimages%2Ffooter-logo.svg&w=32&q=75

### Additional Comments

_No response_

### Contributing

No

### PR DESCRIPTION
**Pull request by quantumflo**
**Issue:** closes #9379

**Similarweb rank:**
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2523,6 +2523,11 @@
             "hex": "FFFFFF",
             "source": "https://clarivate.com"
         },
+        {
+            "title": "Clerk",
+            "hex": "000000",
+            "source": "https://clerk.com/_next/image?url=%2Fimages%2Ffooter-logo.svg&w=32&q=75"
```
Comment (adamrusted) [inline _data/simple-icons.json:2529]:
```suggestion
            "hex": "6C47FF",
            "source": "https://clerk.com"
```

Hex code is available as the primary colour of the buttons etc on the site.
URL-wise we can get away with just the homepage.

File: icons/clerk.svg
Comment (adamrusted) [inline icons/clerk.svg:1]: It looks like the icon hasn't been resized to fit the 24*24 viewport, and still contains additional data (linearGradient, etc).
Could you scale the icon  down to sit in the center of the 24*24 viewport in any vector editing software, and then run `npx svgo icons/clerk.svg` to remove any unnecessary properties - for example, the linearGradient.

**Review by adamrusted — CHANGES_REQUESTED** [review]
Hey @quantumflo - thanks so much for your contribution!
I can see the linter is having a few errors with the SVG, and I've got a few changes to the JSON code which I'd suggest.
If you need any help adjusting the SVG, let me know and I should be able to help you out.

File: icons/clerk.svg
Comment (quantumflo) [reply icons/clerk.svg:1]: I have fixed it now

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for the quick turnaround @quantumflo!
Looks like the icon has been stretched in both directions to fill the 24x24 viewport - whereas it should only be the longest side, with the aspect ratio maintained. See below for an example using the header logo against your PR.
image


Would you mind having another look at this one? 🙏

**Review by adamrusted — CHANGES_REQUESTED** [review]
Still seeing [minor differences](https://github.com/simple-icons/simple-icons/assets/1830867/3f9a8dcd-b934-4d94-b947-600d12b1623f) between source and PR. Might be best to regenerate from source with this one @quantumflo.

**Comment by quantumflo** [comment]
> Still seeing [minor differences](https://github.com/simple-icons/simple-icons/assets/1830867/3f9a8dcd-b934-4d94-b947-600d12b1623f) between source and PR. Might be best to regenerate from source with this one @quantumflo.

Hi I would need help on this. As I am not sure on what changes needs to be done further... @adamrusted

**Comment by quantumflo** [comment]
> > Still seeing [minor differences](https://github.com/simple-icons/simple-icons/assets/1830867/3f9a8dcd-b934-4d94-b947-600d12b1623f) between source and PR. Might be best to regenerate from source with this one @quantumflo.
> 
> Hi I would need help on this. As I am not sure on what changes needs to be done further... @adamrusted

@adamrusted What needs to be done next? I am trying to use inkspace to fix the issues and steps I followed was -
1) scaled it to vertically(longest side )  and then aligned it to center.

**Comment by adamrusted** [comment]
Weird - it shouldn't be stretching that much when you edit the file.
Below is the path I've been using to compare against, feel free to use that. It _should_ be optimised and match up.
```xml
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Clerk</title><path d="m21.47 20.829-2.881-2.881a.572.572 0 0 0-.7-.084 6.854 6.854 0 0 1-7.081 0 .576.576 0 0 0-.7.084l-2.881 2.881a.576.576 0 0 0-.103.69.57.57 0 0 0 .166.186 12 12 0 0 0 14.113 0 .58.58 0 0 0 .239-.423.576.576 0 0 0-.172-.453Zm.002-17.668-2.88 2.88a.569.569 0 0 1-.701.084A6.857 6.857 0 0 0 8.724 8.08a6.862 6.862 0 0 0-1.222 3.692 6.86 6.86 0 0 0 .978 3.764.573.573 0 0 1-.083.699l-2.881 2.88a.567.567 0 0 1-.864-.063A11.993 11.993 0 0 1 6.771 2.7a11.99 11.99 0 0 1 14.637-.405.566.566 0 0 1 .232.418.57.57 0 0 1-.168.448Zm-7.118 12.261a3.427 3.427 0 1 0 0-6.854 3.427 3.427 0 0 0 0 6.854Z"/></svg>
```

**Comment by quantumflo** [comment]
> Weird - it shouldn't be stretching that much when you edit the file. Below is the path I've been using to compare against, feel free to use that. It _should_ be optimised and match up.
> 
> ```
> <svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Clerk</title><path d="m21.47 20.829-2.881-2.881a.572.572 0 0 0-.7-.084 6.854 6.854 0 0 1-7.081 0 .576.576 0 0 0-.7.084l-2.881 2.881a.576.576 0 0 0-.103.69.57.57 0 0 0 .166.186 12 12 0 0 0 14.113 0 .58.58 0 0 0 .239-.423.576.576 0 0 0-.172-.453Zm.002-17.668-2.88 2.88a.569.569 0 0 1-.701.084A6.857 6.857 0 0 0 8.724 8.08a6.862 6.862 0 0 0-1.222 3.692 6.86 6.86 0 0 0 .978 3.764.573.573 0 0 1-.083.699l-2.881 2.88a.567.567 0 0 1-.864-.063A11.993 11.993 0 0 1 6.771 2.7a11.99 11.99 0 0 1 14.637-.405.566.566 0 0 1 .232.418.57.57 0 0 1-.168.448Zm-7.118 12.261a3.427 3.427 0 1 0 0-6.854 3.427 3.427 0 0 0 0 6.854Z"/></svg>
> ```

Cool thanks. Updated now

**Comment by quantumflo** [comment]
@adamrusted this PR can be merged?

**Comment by quantumflo** [comment]
> @adamrusted this PR can be merged?

Hey @adamrusted any other changes required?

**Comment by adamrusted** [comment]
Apologies @quantumflo, I'm not able to review the PR given the path I contributed has been used. @simple-icons/icon-review - anyone able to give this a once over?

**Review by PratyushJayachandran — APPROVED** [review]
Perfect match @adamrusted! Thanks for starting the contribution @quantumflo 🎉

**Pull request merged**